plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -6902,6 +6903,260 @@ public void testNoRetryWhenMetadataFileInvalid()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testPartitionFilterRemoved()
+    {
+        String randomSuffix = UUID.randomUUID().toString().replace("-", "");
+        String tableName = "partition_test_" + randomSuffix;
+
+        Session session = Session.builder(getQueryRunner().getDefaultSession())
+                .setCatalogSessionProperty("iceberg", "query_partition_filter_required", "true")
+                .build();
+
+        assertUpdate(
+                session,
+                "create table " + tableName + "(\n"
```

Comment:
Uppercase "create table" and other reserved keywords. Remove redundant `\n`. Same for others